test/index.js

```diff
@@ -1,40 +1,113 @@
+      var phase = lune.phase(new Date("2014-02-17T00:00-0500"));
 
-      var feb17 = {
-        phase: 0.5616632223402672,
-        illuminated: 0.9629393807872504,
-        age: 16.586245595613818,
-        distance: 396868.3763643785,
-        angular_diameter: 0.5018242066159135,
-        sun_distance: 147816061.66410872,
-        sun_angular_diameter: 0.5395080276270386
-      };
+      assert.closeTo(phase.phase,                    0.568 ,      0.001 );
+      assert.closeTo(phase.illuminated,              0.955 ,      0.001 );
+      assert.closeTo(phase.age,                     16.779 ,      0.030 );
+      assert.closeTo(phase.distance,            396084.5   ,    384.4   );
+      assert.closeTo(phase.angular_diameter,         0.5028,      0.0005);
+      assert.closeTo(phase.sun_distance,     147822500     , 149600     );
+      assert.closeTo(phase.sun_angular_diameter,     0.5395,      0.0005);
+    });
 
-      var lunedata = lune.phase(new Date(2014, 1, 17));
+    // http://bazaar.launchpad.net/~keturn/py-moon-phase/trunk/view/head:/moontest.py
+    it("should be accurate to astronomical observations", function() {
+      var observations = [
+            ["1989-01-07T19:22Z", 0.00],
+            ["1989-01-14T13:58Z", 0.25],
+            ["1989-01-21T21:33Z", 0.50],
+            ["1989-01-30T02:02Z", 0.75],
+            ["1989-02-06T07:37Z", 0.00],
+            ["1989-02-12T23:15Z", 0.25],
+            ["1989-02-20T15:32Z", 0.50],
+            ["1989-02-28T20:08Z", 0.75],
+            ["1989-03-07T18:19Z", 0.00],
+            ["1989-03-14T10:11Z", 0.25],
+            ["1989-03-22T09:58Z", 0.50],
+            ["1989-03-30T10:21Z", 0.75],
+            ["1989-04-06T03:33Z", 0.00],
+            ["1989-04-12T23:13Z", 0.25],
+            ["1989-04-21T03:13Z", 0.50],
+            ["1989-04-28T20:46Z", 0.75],
+            ["1989-05-05T11:46Z", 0.00],
+            ["1989-05-12T14:19Z", 0.25],
+            ["1989-05-20T18:16Z", 0.50],
+            ["1989-05-28T04:01Z", 0.75],
+            ["1989-06-03T19:53Z", 0.00],
+            ["1989-06-11T06:59Z", 0.25],
+            ["1989-06-19T06:57Z", 0.50],
+            ["1989-06-26T09:09Z", 0.75],
+            ["1989-07-03T04:59Z", 0.00],
+            ["1989-07-11T00:19Z", 0.25],
+            ["1989-07-18T17:42Z", 0.50],
+            ["1989-07-25T13:31Z", 0.75],
+            ["1989-08-01T16:06Z", 0.00],
+            ["1989-08-09T17:28Z", 0.25],
+            ["1989-08-17T03:07Z", 0.50],
+            ["1989-08-23T18:40Z", 0.75],
+            ["1989-08-31T05:44Z", 0.00],
+            ["1989-09-08T09:49Z", 0.25],
+            ["1989-09-15T11:51Z", 0.50],
+            ["1989-09-22T02:10Z", 0.75],
+            ["1989-09-29T21:47Z", 0.00]
+          ];
 
-      assert.equal(JSON.stringify(feb17), JSON.stringify(lunedata));
+      var error = 0;
+      var i;
+      var obs;
+      var e;
+      for(i = 0; i < observations.length; i++) {
```

Comment:
Oooh, yeah let's do that. (I haven't been keeping up with all the changes Node has been making). You can bump the Node version in the travis yaml file. Ideally we test against the latest stable Node version as well as the LTS version.